recipes/macos.rb

```diff
@@ -1,3 +1,44 @@
+## Underautomated setup instructions
+# This section lists operations which were resistant to initial attempts to
+# automate them.
+# Hopefully with time, effort, and documentation we can find ways to automate these steps as well.
+
+# Start by checking for updates and running any pending OS updates.
+# Do not do major macOS version upgrades, such as Monterey -> Ventura.
+
+# `administrator` user should already exist and the password is available in Bitwarden.
+
+# Create `jenkins` account with sudo / administrator access to the host.
+# Set it up for passwordless sudo.
+
+# Log in as the Jenkins user, leaving accessibility, siri, and apple ID sign in disabled during initial user setup.
+
+# Enable autologin for Jenkins from Login options, this is required so that xquartz is started on system boot.
+
+# Verify SSH and VNC remote access are enabled, which should already true for
+# our hosted machines.
+# Verify remote management is enabled _only_ for administrator
+
+# Verify wifi and bluetooth are disabled, which should already be true for
+# our hosted machines.
+
+# Disable Spotlight indexing. It's worth doing globally but at the very least
```

Comment:
[This](https://github.com/Microsoft/macos-cookbook/blob/master/documentation/resource_spotlight.md) microsoft chef resource could be useful for the automation of this.

Reply:
Agreed! I am strongly committed to avoiding cookbook dependencies but this one may be worth adopting in order to deal more directly with macos intricacies that are hard to model within our own recipes. However, I think that merging this as-is and automating more of the set up in further PRs is the way to go.